pkg/kubelet/dockershim/libdocker/helpers_test.go

```diff
@@ -51,8 +51,8 @@ func TestMatchImageTagOrSHA(t *testing.T) {
 			Output:    true,
 		},
 		{
-			Inspected: dockertypes.ImageInspect{RepoTags: []string{"docker.io/kubernetes/pause:latest"}},
-			Image:     "kubernetes/pause:latest",
+			Inspected: dockertypes.ImageInspect{RepoTags: []string{"k8s.gcr.io/pause:3.1"}},
```

Comment:
hm, actually, maybe we should have left this one? I'm not sure what exactly this test is doing, but the docker.io bit may have been relevant here.

Reply:
I was also a little hesitant on this.

Kindly ping @yujuhong: do you see any side effect of this?

Reply:
Please don't change this. The test verifies that the two match even with the `docker.io` difference.

Reply:
@yujuhong thanks for clarifying! I will keep it as is.